eth-contracts/contracts/Governance.sol

```diff
@@ -379,6 +394,9 @@ contract Governance is InitializableV2 {
             "Governance: To submit new vote, call submitVote()"
         );
 
+        // Validates new _vote, _proposalId, proposal state, and voter state + returns voterStake
+        uint256 voterStake = proposals[_proposalId].voteMagnitude[voter];
```

Comment:
expose voteMagnitude property for a given proposalId and voter

Reply:
done